journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedKeyJournal.scala

```diff
@@ -30,6 +30,11 @@ trait ReplicatedKeyJournal[F[_]] {
     offset: Offset,
     timestamp: Instant,
   ): F[Changed]
+
+  def expire(
```

Comment:
I'd add a single sentence scaladoc explaining what does it do, smth like "Updates expiration time for a key". Otherwise looks legit

Reply:
If you add, you are free to resolve and merge it yourself, I have no objections.